src/Database/DBStructure.php

```diff
@@ -1733,7 +1733,8 @@ public static function definition() {
 						"page-flags" => ["type" => "tinyint", "not null" => "1", "default" => "0", "comment" => ""],
 						"account-type" => ["type" => "tinyint", "not null" => "1", "default" => "0", "comment" => ""],
 						"prvnets" => ["type" => "boolean", "not null" => "1", "default" => "0", "comment" => ""],
-						"pwdreset" => ["type" => "varchar(255)", "not null" => "1", "default" => "", "comment" => ""],
+						"pwdreset" => ["type" => "varchar(255)", "comment" => "Password reset request token"],
```

Comment:
When you change the database structure, please always execute this command:
```
scripts/dbstructure.php dumpsql > database.sql
```

Reply:
Done, thanks for the reminder.